app/views/devise/passwords/edit.html.erb

```diff
@@ -11,19 +11,19 @@
             <%= f.hidden_field :reset_password_token %>
 
             <div class="field">
-                <%= f.label :password, "New password" %><br />
+                <%= f.label :password, t('.new_password') %><br />
                 <% if @minimum_password_length %>
-                    <em>(<%= @minimum_password_length %> characters minimum)</em>
+                    <em>(<%= t('.min_characters', min_chars: @minimum_password_length ) %>)</em>
                 <% end %>
                 <%= f.password_field :password, autofocus: true, autocomplete: "new-password" %>
             </div>
 
             <div class="field">
-                <%= f.label :password_confirmation, "Confirm new password" %><br />
+                <%= f.label :password_confirmation, t('.confirm_new_password') %><br />
                 <%= f.password_field :password_confirmation, autocomplete: "off" %>
             </div>
 
-            <%= f.submit "Change my password", class: "ui button primary" %>
+            <%= f.submit class: "ui button primary" %>
```

Comment:
We can omit value here 'cause it taken from default I18n keys for model.
![image](https://user-images.githubusercontent.com/7688888/68277140-62bc0000-0090-11ea-9e2c-b3f87e97e760.png)

Reply:
I am not sure it's a good idea since it's kinda unexpected without [this](https://github.com/rails/rails/blob/v5.0.7.2/actionview/lib/action_view/helpers/form_helper.rb#L1813). For me it seems like a hack and it encourages using it in that way.

```
def test(one = nil, two = {})
	p one
	p two
end
```

Result:
```
test(class: 'test')
p RUBY_VERSION
```

Notice that `class` was assigned to param `one`, not `two`.

Reply:
Heh, I suppose it's more of semantic sugar than hack, and if it's implemented in framework it's legit AF... but I'll fix it anyway for the sake of consistency.